### ISSUE TITLE
Linux OSX support?

### ISSUE THREAD

**Issue by eix128** [microsoft/TSS.MSR#170]
Hi can this library work on Linux and OSX ?

**Comment by nd1012** [comment]
Yes, if there's a TPM2 available. Apple uses its own T2 chip in many devices, which is similar to TPM2 - but not usable with this library.
